src/riak_kv_replrtq_snk.erl

```diff
@@ -339,25 +381,39 @@ calc_mean(Time, Count) ->
 % usable peer information.
 % tokenised string expected to be of form:
 % "192.168.10.1:8098|192.168.10.2:8098 etc"
```

Comment:
should we not have a name here in the comment as well?

Reply:
Updated comment